packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -90,6 +93,34 @@ abstract class LambdaAuthorizer extends Authorizer implements IAuthorizer {
     }
 
     this.restApiId = restApi.restApiId;
+
+    let functionName;
```

Comment:
is there a reason to do any of this if the feature flag is disabled?

Reply:
Moved